ui/pages/confirmations/confirm/confirm.stories.tsx

```diff
@@ -1,27 +1,92 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import configureStore from '../../../store/store';
+import { cloneDeep } from 'lodash';
+import { unapprovedPersonalSignMsg, signatureRequestSIWE } from '../../../../test/data/confirmations/personal_sign';
+import { unapprovedTypedSignMsgV1, unapprovedTypedSignMsgV4, permitSignatureMsg } from '../../../../test/data/confirmations/typed_sign';
 import mockState from '../../../../test/data/mock-state.json';
+import configureStore from '../../../store/store';
 import ConfirmPage from './confirm';
 
-const store = configureStore({
-  confirm: {
-    currentConfirmation: {
-      type: 'personal_sign',
-    },
-  },
-  metamask: {
-    ...mockState.metamask,
-  },
-});
-
 const ConfirmPageStory = {
   title: 'Pages/Confirm/ConfirmPage',
-  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+  decorators: [(story) => <div style={{ height: '600px' }}>{story()}</div>],
```

Comment:
Should we also force the width so it matches the standard MetaMask popup?

Reply:
Good question. As you might have noticed, the height resembles our popup height. I added this to view the scrollToBottom feature on some of the signature pages without needing to hardcode or add a control for this.  

I left out the width so that we could easily test various widths. These decisions were subjective. Open to other suggestions

![CleanShot 2024-06-07 at 21 21 04](https://github.com/MetaMask/metamask-extension/assets/20778143/9b60f1cb-0435-4d6d-a0f8-5d7764a6e2b6)

Reply:
Could we still support re-sizing to test wrapping etc, but default to the standard width?